common/core/web/keyboard-processor/src/keyboards/keyboard.ts

```diff
@@ -98,24 +98,27 @@ namespace com.keyman.keyboards {
     }
 
     /**
-     * HTML help text which is a one liner intended for the status bar of the desktop OSK originally.
+     * HTML help text, as specified by either the &kmw_helptext or &kmw_helpfile system stores.
      * 
-     * Reference: https://help.keyman.com/developer/language/reference/kmw_helptext
+     * Reference: https://help.keyman.com/developer/language/reference/kmw_helptext,
+     *            https://help.keyman.com/developer/language/reference/kmw_helpfile
      */
     get helpText(): string {
       return this.scriptObject['KH'];
     }
 
-    get hasHelpHTML(): boolean {
+    get hasScript(): boolean {
```

Comment:
Please add a comment, similar to the one above for `helpText`, referencing kmw_embedjs